pkg/sypgp/sypgp.go

```diff
@@ -349,6 +350,68 @@ func StorePubKey(e *openpgp.Entity) (err error) {
 	return
 }
 
+// compareLocalPubKey : compares a key ID with a string. return true if the
```

Comment:
Usually godoc strings should read like a sentence (so that when formatting them using go doc it comes out nicely).

For example:

    // compareLocalPubKey compares a key ID with a string, returning true if the
    // key and oldToken do not match.

Also, because of the name of the function, I would expect it to return true if they do match. Please consider changing the name of the function (or inverting the sense of the return value so that it matches the name of the function).

Reply:
On it! 
And yes, I know the true/false is backwards... fixing that.